examples/pattern_rewriting.py

```diff
@@ -67,45 +67,27 @@ def get_rotary_model(bad_model=False):
 # The rewriting pattern
 # =====================
 
-op = onnxscript.opset18
-msft_op = onnxscript.values.Opset("com.microsoft", 1)
 
-
-def rotary_match_pattern(x, pos_ids, axis):
+def rotary_match_pattern(op, x, pos_ids, axis):
     """The pattern to match."""
     unsqueeze = op.Unsqueeze(x, axis)
     cast = op.Cast(unsqueeze, to=onnx.TensorProto.FLOAT)
 
     matmul = op.MatMul(pos_ids, cast)
     transpose = op.Transpose(matmul)
-    output, length = msft_op.ConcatTraining(transpose, transpose)
+    output, length = op.ConcatTraining(transpose, transpose, domain="com.microsoft", outputs=2)
```

Comment:
The IR uses `outputs` to accept a list of Values and num_outputs for an int. Thoughts on consistency?

Reply:
Hmmm ... I used to have `_num_outputs` (in building a pattern) ... I switched to `outputs` to match what was there (after the update to generic-matcher to use the new IR ... not sure what the origin of this was)!

I think that the overloaded use of `outputs` to denote either the number of outputs or names of outputs is reasonable ... we could use distinct names too, but then would have to check that they are not used inconsistently. Typically, we expect only one or the other to be used.

But whatever we decide, agree we should have a consistent one.